services/MessagingService.php

```diff
@@ -35,15 +35,15 @@ public function processNotification(BaseNotification $baseNotification, User $us
         );
     }
 
-    public function processMessage(User $user, string $title, string $body, ?string $url, ?string $imageUrl, ?int $notificationCount)
+    public function processMessage(User $user, ?string $title, ?string $body, ?string $url, ?string $imageUrl, ?int $notificationCount)
```

Comment:
@yurabakhtin Is it really a good idea to allow notifications without any text?

Reply:
@luke- I don't know why the error exists at all because the params cannot be null:

- The notification `humhub\modules\comment\notifications\NewComment` cannot return `null` as result of the method `text()`, because [here](https://github.com/humhub/humhub/blob/master/protected/humhub/components/SocialActivity.php#L273-L278) the method must return some string if the method `html()` returns some not empty string,
- the method `NewComment->html()` doesn't return `null`  - https://github.com/humhub/humhub/blob/master/protected/humhub/modules/comment/notifications/NewComment.php#L169-L187, even if a comment is deleted,
- so it is strange why [here](https://github.com/humhub/fcm-push/blob/master/services/MessagingService.php#L31) `$baseNotification->text()` returns `null`.

Reply:
@yurabakhtin That's really strange. Maybe `html_entity_decode` returns null? In any case, I would not recommend the current PR but rather look for the cause.